bin/lint

```diff
@@ -19,6 +19,7 @@ fi
 
 run check-required-files
 node_modules/.bin/eslint \
+  --no-error-on-unmatched-pattern \
   --report-unused-disable-directives \
   -- "${files[@]}" '*.{js,mjs}' 'test/**/*.{js,mjs}'
```

Comment:
By specifying [`--no-error-on-unmatched-pattern`][1] it is no longer necessary for each of these globs to match at least one file. In the past I have resorted to creating empty files in projects without tests.


[1]: https://eslint.org/docs/user-guide/command-line-interface#no-error-on-unmatched-pattern